src/modal/test/modal.spec.js

```diff
@@ -73,9 +73,11 @@ describe('$modal', function () {
       toHaveModalOpenWithContent: function(util, customEqualityTesters) {
         return {
           compare: function(actual, content, selector) {
-            var contentToCompare, modalDomEls = actual.find('body > div.modal > div.modal-dialog > div.modal-content');
+            var container = (actual === $document) ? actual.find('body') : angular.element(actual);
```

Comment:
Shouldn't `actual` already be a jqLite wrapped element? I believe `angular.element(actual)` should just be `actual` here.